.env.example

```diff
@@ -1,19 +1,20 @@
 APP_NAME=Laravel
 APP_ENV=local
-APP_KEY=
+APP_KEY=base64:UEwoIo/1JRhMBGAd/qTDOX+E2Tx+fZ7ifBPHUzyFG7w=
```

Comment:
This is not necessary at all, reason behind adding it in .env.example file is when we 'cp .env.example .env' , it comes into .env file, otherwise we need to generate key, but when I was setting up and testing application I added that in this file.

Reply:
The readme include `php artisan key:generate` in the docker command. Have you run that docker command as it doesn't work as is?